examples/CIB/ex1/main.cpp

```diff
@@ -399,6 +495,83 @@ main(int argc, char* argv[])
             pout << "+++++++++++++++++++++++++++++++++++++++++++++++++++\n";
             pout << "\n";
 
+            // Fill ghost cells of pressure and velocity to compute hydrodynamic forces.
```

Comment:
@nishantnangia : Create a function in IBHydrodynamicForceEvaluator that does this ghost-filling of cloned velocity and pressure variables. Make it virtual so that users can override the default behavior.

@boyceg : We are not creating variables inside this class for efficiency reasons. If the control volume does not touch computational domain boundary once can just fill ghost cells of current velocity and pressure variable straight out of INSIntegrator and do these calculations. Right now, we are creating additional variables that also fill ghost cells at the domain boundary. Are you OK with the design of creating cloned variables in main.cpp but filling them inside force evaluator class?